packages/suite/src/types/coinmarket/coinmarketOffers.ts

```diff
@@ -0,0 +1,123 @@
+import { Account } from '@suite-common/wallet-types';
+import { AppState } from '../suite';
+import { ExchangeInfo } from 'src/actions/wallet/coinmarketExchangeActions';
+import {
+    BankAccount,
+    BuyTrade,
+    CryptoSymbol,
+    ExchangeTrade,
+    P2pProviderInfo,
+    P2pQuote,
+    P2pQuotesRequest,
+    SellFiatTrade,
+} from 'invity-api';
+import { Timer } from '@trezor/react-utils';
+import { BuyInfo } from 'src/actions/wallet/coinmarketBuyActions';
+import { UseCoinmarketFilterReducerOutputProps } from 'src/reducers/wallet/useCoinmarketFilterReducer';
+import { TradeSell } from '../wallet/coinmarketCommonTypes';
+import { SellInfo } from 'src/actions/wallet/coinmarketSellActions';
+import { CoinmarketTradeBuyType, CoinmarketTradeSellType, CoinmarketTradeType } from './coinmarket';
+
+type CoinmarketOffersContextProps = {
+    type: CoinmarketTradeType;
+    device: AppState['device']['selectedDevice'];
+    account: Account;
+    callInProgress: boolean;
+    timer: Timer;
+    getQuotes: () => Promise<void>;
+    selectQuote: (quote: BuyTrade | SellFiatTrade | ExchangeTrade) => void;
+};
+
+type CoinmarketBuyOffersContextProps = CoinmarketOffersContextProps & {
+    quotesRequest: AppState['wallet']['coinmarket']['buy']['quotesRequest'];
+    quotes: AppState['wallet']['coinmarket']['buy']['quotes'];
+    selectedQuote?: BuyTrade;
+    verifyAddress: (account: Account, address?: string, path?: string) => Promise<void>;
+    addressVerified: AppState['wallet']['coinmarket']['buy']['addressVerified'];
+    providersInfo?: BuyInfo['providerInfos'];
```

Comment:
shouldn't be there `buyInfo?: BuyInfo;` just like in Sell and Exchange contextes? not sure if it's needed but seems a bit inconsistent like this